tutorials/welding_example_02_weaving.ipynb

```diff
@@ -531,7 +531,9 @@
    "outputs": [],
    "source": [
     "t_interp = pd.timedelta_range(\n",
-    "    start=tcp_wire.time[0].values, end=tcp_wire.time[-1].values, freq=\"20ms\"\n",
+    "    start=tcp_wire.time.as_pandas_index()[0], \n",
+    "    end=tcp_wire.time.as_pandas_index()[-1], \n",
+    "    freq=\"20ms\"\n",
```

Comment:
can you try `tcp_wire.time[0]` here now with `__getitem__` @vhirtham ?

Reply:
See commit 5dcd477888d035797bda162586763ff758876166 😉

Reply:
nice :)